nemesis/arena_prep.py

```diff
@@ -588,6 +590,7 @@ def advanced_battle_logic():
 
 
 def bribe_battle(spare_change_value: int = 12000):
+    print("value (2): ", spare_change_value)
```

Comment:
This should either be logging out removed

Reply:
nemesis needs to be updated across the board anyway. Not worried about this.